packages/contracts-bedrock/contracts/L2/GasPriceOracle.sol

```diff
@@ -42,7 +42,7 @@ contract GasPriceOracle is Ownable, Semver {
     /**
      * @notice Number of decimals used in the scalar.
      */
-    uint256 public decimals;
+    uint256 public decimals = 6;
```

Comment:
Might as well make this a `constant`

Reply:
If we make it a constant, will it be in the bytecode or in the state?

Reply:
It must be in the state for this to be backwards compatible

Reply:
It appears as if making it a constant does not place it in state, so we cannot do this unfortunately

Reply:
what is the backwards compatibility issue?

Reply:
The decimals are read from state in the L2 geth state transition function

Reply:
as is written, I believe that this syntax depends on running initcode to set the value...

Reply:
To clarify, the same scheme is use in the bedrock optigeth as old l2geth

Reply:
Got it. Let's leave it in state then.